packages/eslint-plugin/ROADMAP.md

```diff
@@ -112,7 +112,7 @@ It lists all TSLint rules along side rules from the ESLint ecosystem that are th
 | TSLint rule                  |     | ESLint rule                                        |
 | ---------------------------- | :-: | -------------------------------------------------- |
 | [`cyclomatic-complexity`]    | 🌟  | [`complexity`][complexity]                         |
-| [`deprecation`]              | 🌓  | [`import/no-deprecated`] <sup>[1]</sup>            |
```

Comment:
you need to add this link to the bottom section.

Reply:
But it does not apply anymore, since the rules applies not only to imports but to all deprecated references just like original tslint rule...

Reply:
You need to add the link to the bottom section so that this line renders as a link

![image](https://user-images.githubusercontent.com/7462525/74690240-2d48a780-5193-11ea-94f3-b70d93e4664f.png)

Bottom section:
https://github.com/gund/typescript-eslint/blob/patch-1/packages/eslint-plugin/ROADMAP.md#L677-L688

Reply:
Oh I see, will update

Reply:
Done